src/services/user.js

```diff
@@ -505,6 +504,141 @@ const updateUser = async (req, res, next) =>
     }
 }
 
+const updateUserToAdmin = async (req, res, next) => {
+    try{
+
+        let userEmail = req.path.split('/');
+        userEmail = userEmail[userEmail.length - 1];
+
+        Logger.write('user', `Update UserToAdmin with email  ${userEmail}`);
+
+        let user = await User.findOne({'email': userEmail});
+        const userId = user._id;
+
+        let isUserExists = await User.findById(userId);
+
+        if (!isUserExists) {
+            return res.status(404).json({
+                'code': 'BAD_REQUEST_ERROR',
+                'description': 'No user found in the system'
+            });
+        }
+
+        const updateAdminStatus = {
+            isAdmin : true
+        }
+
+        let updateUser = await User.findByIdAndUpdate(userId, updateAdminStatus, {
```

Comment:
@ayojava I think we might have to check the admin previlleges of the user. Otherwise an student who has the login and who got to know this endpoint they will be able to make them selves admin...

Reply:
It has been fixed